CWMS/cwms_level.py

```diff
@@ -67,16 +62,14 @@ def delete_specified_level(self, specified_level_id: str, office_id: str):
             raise ValueError("Cannot delete a specified level without an office id")
         end_point = f"{CwmsLevel._SPECIFIED_LEVELS_ENDPOINT}/{specified_level_id}"
 
-        params = {
-            OFFICE_PARAM: office_id
-        }
-        headers = {
-            "Content-Type": HEADER_JSON_V2
-        }
+        params = {OFFICE_PARAM: office_id}
+        headers = {"Content-Type": HEADER_JSON_V2}
         response = self.get_session().delete(end_point, params=params, headers=headers)
         response.raise_for_status()
 
-    def update_specified_level(self, old_specified_level_id: str, new_specified_level_id: str, office_id: str):
+    def update_specified_level(
+        self, old_specified_level_id: str, new_specified_level_id: str, office_id: str
```

Comment:
Is there a divergent style guide you're using for this? I'm following the PEP 8 E501 for line length of 120

Reply:
yeah VS code is formatting that using prettier.  I think I like your better.  I didn't notice the change until I created the pull request.  I will try to update my vs code.

Reply:
I haven't seen an existing HEC style guide for python so I've just defaulted to whatever PEP 8 flags that IntelliJ tells me

Reply:
I am using autopep8 from VS code.  it automates the formatting.  I will likely stick with this format.